effect/mocks/executor.go

```diff
@@ -3,8 +3,9 @@
 package mocks
 
 import (
-	effect "github.com/paketo-buildpacks/libpak/effect"
```

Comment:
This is a generated file that's going to be reverted each time we regenerate the file.  Probably shouldn't update it as part of this PR.